core-galleon-pack/src/main/resources/layers/standalone/management/layer-spec.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="management">  
+    <dependencies>
+        <layer name="base-server"/> 
+    </dependencies>
+    <feature-group name="management-interface"/>
+    <feature-group name="management-sockets-group"/>
+    <!-- management layer comes with security -->
+    <feature-group name="access-control"/>
```

Comment:
This use of access-control seems a bit odd too. RBAC is about authorizing an authenticated user, so if the user isn't authenticated it doesn't serve much purpose.

Reply:
It is a way to bring the feature-in without defining a layer. Seemed to make sense to colocate with management.